Doc/library/calendar.rst

```diff
@@ -341,7 +341,7 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
 
 .. note::
 
-   The constructor, :meth:`formatweekday` and :meth:`formatmonthname` methods
+   The constructor, :meth:`!formatweekday` and :meth:`!formatmonthname` methods
```

Comment:
Ignoring valid warnings will only make us forget about the issue. We already have too many open and forgotten issues.

If the goal is to suppress warnings, just add the corresponding option to Sphinx invocation.